pages/features/internationalization.mdx

```diff
@@ -0,0 +1,43 @@
+import { ImageTabs } from '../../components/ImageTabs';
+
+Start UI already implements internationalization, which means that it is designed to support multiple languages.
+It currently supports French, English, Arabic, and Swahili, and is set up to display text in these languages in a way that is appropriate
+for the language's writing system. For example, if you are using Start UI in Arabic, the text will be displayed right-to-left, as is standard for Arabic.
+Similarly, if you are using Start UI in Swahili, the text will be displayed left-to-right, as is standard for languages that use the Latin alphabet.
+This allows users to access the user interface in their preferred language, making it more accessible and user-friendly for people around the world.
+The default language is English.
+
+## Choosing language
+
+To change your selected language, simply go to your profile's page, and select the language you want applied through the **Language** Select :
+
+import selectLanguageDesktop from '../../public/images/screenshots/internationalization/select-language-desktop.png';
+import selectLanguageMobile from '../../public/images/screenshots/internationalization/select-language-mobile.png';
+
+<ImageTabs
+    desktopImage={{
+        alt: "Select language screenshot on desktop",
+        src: selectLanguageDesktop,
+    }}
+    mobileImage={{
+        alt: "Select language screenshot on mobile",
+        src: selectLanguageMobile,
+    }}
+/>
+
+Switching from one language that's written left-to-right to another that is not will automatically apply the expected change after clicking **Save** :
```

Comment:
you said "left to right" and put a screen for arabic, which is right to left

Reply:
I think you misunderstood my statement. Since the previous screenshot is in English, I am trying to showcase that switching from English (which is read left-to-right) to some other language that is not read left-to-right, will trigger a change in the text's writing pattern

Reply:
Sorry, this comment was for this sentence "For example, if you are using Start UI in Arabic, the text will be displayed right-to-left, as is standard for Arabic." but idk why it's here. I think you already changed the previous sentence, it's ok